scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -225,5 +232,47 @@ task ExtractTask {
      }
  }
 
+task GetLastModifiedDateOrUniqueValue {
+    # because this is being used to determine if the data has changed, never use call cache
+    meta {
+        volatile: true
+    }
+
+    input {
+        String data_project
+        String default_dataset
+        String dataset_table
+        File? service_account_json
+    }
+
+    String has_service_account_file = if (defined(service_account_json)) then 'true' else 'false'
+
+    # ------------------------------------------------
+    # try to get the last modified date for the table in question; if something comes back from BigQuwey
+    # that isn't in the right format (e.g. an error), throw it away and generate something unique
+    command {
+        if [ ~{has_service_account_file} = 'true' ]; then
```

Comment:
Meta question: do we not have a base docker image that already has service account mounted?  That's totally bigger than this PR but if this is something we do a lot, it seems like it could be useful?

Reply:
we don't -- It's generally not a good practice to bake credentials into a docker image in case it's compromised, but also these service account keys are rotated frequently so we need to get a new one when we run

Reply:
Yeah, I wasn't thinking we'd bake the SA into the image but rather mount it in a volume in a standard location